fe/fe-core/src/main/java/com/starrocks/mysql/security/LdapSecurity.java

```diff
@@ -47,7 +47,8 @@ public static boolean checkPassword(String dn, String password) {
             ctx = new InitialDirContext(env);
             return true;
         } catch (Exception e) {
-            LOG.warn("check ldap password failed, dn = {}, password = {}", dn, password, e);
+            LOG.warn("check ldap password failed, dn = {}", dn, e);
+            LOG.debug("check ldap password failed, dn = {}, password = {}", dn, password);
```

Comment:
why still log the password? is it necessary even in debug log level?

Reply:
I think we can't print credential information in the log.